index.bs

```diff
@@ -1,317 +1,215 @@
 		}
 		</style>
 
-		<div class=locked>
+		<div class=rs>
 		  ... some content goes here ...
 		</div>
 	</xmp>
 
-	The .locked element's 'render-subtree' configuration
-	lets the user-agent manage rendering the subtree of the element.
-	Specifically when this element is near the viewport,
-	the user-agent will begin rendering the element.
-	When the element moves away from the viewport,
-	it will stop being rendered.
-
-	Recall that when not rendered,
-	the property also applies [=size containment=] to the element.
-	This means that when not rendered,
-	the element will use the specified 'contain-intrinsic-size',
-	making the element layout as if it had a single block child with 100px width and 200px height.
-	This ensures that the element still occupies space when not rendered.
-	At the same time,
-	it lets the element size to its true contents when the subtree is rendered
-	(since [=size containment=] no longer applies),
-	thus removing the concern that estimates like 100x200 are sometimes inaccurate
-	(which would otherwise result in displaying incorrect layout for on-screen content).
+  The .rs element's 'render-subtree' value [=hidden=] lets the user-agent manage
+  whether to paint and hit-test the element's subtree.  Specifically when this
+  element is near the viewport, the user-agent will begin painting the element.
+  When the element moves away from the viewport, it will stop being painted. In
+  addition, the user-agent should skip as much of the rendering work as
+  possible when the element is not painted or hit-tested.
 </div>
 
 <div class=example>
 	<xmp highlight=markup>
 		<style>
-		.locked {
-		  render-subtree: invisible skip-activation;
-		  contain-intrinsic-size: 100px 200px;
+		.rs {
+		  render-subtree: hidden frozen;
 		}
 		</style>
 
-		<div class=locked>
+		<div class=rs>
 		  ... some content goes here ...
 		</div>
 	</xmp>
 
-	In this case, the rendering of the subtree is managed by the developer only.
-	This means that if script does not modify the value,
-	the element's subtree will remain unrendered,
-	and it will use the 'contain-intrinsic-size' input when deciding how to size the element.
-</div>
-
-<div class=example>
-	<xmp highlight=markup>
-		<style>
-		.locked {
-		  render-subtree: invisible skip-viewport-activation;
-		}
-		</style>
-
-		<div class=locked>
-		  ... some content goes here ...
-		</div>
-	</xmp>
+  In this case, the element's subtree is not painted or hit-tested regardless
+  of viewport intersection. This means that the only way to have this subtree
+  painted is via script updating the value to remove render-subtree or change
+  its value. As before, the user-agent should skip as much of the rendering in
+  the subtree as possible. An additional effect of skipping rendering is that
+  the layout state of the subtree can be preserved, so that removing the
+  'render-subtree' property in the future will cause the subtree to be rendered
+  quicker than otherwise possible.
```

Comment:
Also add an example where getBoundingClientRect is used together with render-subtree: hidden frozen